website/content/02-guides/02-schema.mdx

```diff
@@ -2,7 +2,7 @@
 title: 'Schema'
 ---
 
-## Schema
+## Overview
 
 [API Reference](/api/modules/main/exports/schema) ⌁ [issues](https://nxs.li/issues/component/schema) / [features](https://nxs.li/issues/components/schema/features) | [bugs](https://nxs.li/issues/component/schema/bugs)
```

Comment:
This can probably just be removed?

Reply:
I have a feature request to allow h1 -> content rather than enforce h1 -> h2 -> content; right now it breaks the design.

Reply:
https://github.com/prisma/docs/issues/522

Reply:
This comment was about line 7 (not the diff) - these links.

Reply:
😳 Clearly I'm stupid today.